testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractEnvSinglenodeLdap.java

```diff
@@ -36,7 +36,7 @@
 import static java.util.Objects.requireNonNull;
 import static org.testcontainers.utility.MountableFile.forHostPath;
 
-public abstract class AbstractSinglenodeLdap
```

Comment:
How about removing `Singlenode` or `Multinode`? This to me is some kind of detail. Then we could have `AbstractLdapEnvironment` which looks better.

Reply:
I would consider this an unrelated change and would prefer to keep it out of scope of this PR. Am i missing something?

Reply:
Are there any environments which NEED to be singlenode? If not let's instead focus the effort on moving everything to Multinode - that way the naming is also simplified.

Reply:
It is unrelated. But if we are going to do a big rename let's do one rename instead of several ones.

> Are there any environments which NEED to be singlenode?

I don't think this is a REAL use case for anything. There is a case where it is possible to use singlenode, but that is not the point of product tests. We should test things in a way they are used. Notice that `io.trino.tests.product.launcher.env.common.Standard` that is used for singlenode is already deprecated.

Reply:
> Are there any environments which NEED to be singlenode?

there might be memory constraint

> If not let's instead focus the effort on moving everything to Multinode - that way the naming is also simplified.

if we succeed in this, then we could want to remove Multinode prefix, which would be noise only

Reply:
> Are there any environments which NEED to be singlenode?

Yes.

- single-node setup should work (e.g. for demo purposes like when running trinodb/trino docker image), and so we should have test coverage for such setup
- single-node setup is easier to debug and makes troubleshooting problems much easier, so it's useful as a developers' configuration (similar to how `DevelopmentServer` is useful)
  - as we move more and more things to multinode, we may want to introduce option to "run this (multinode) environment, but actually as a singlenode", to help debugging.